JanssenProject/jans
# docs(pycloudlib): update docstrings and internal API docs

### Prepare

- [x] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [x] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

### Description

#### Target issue
 
GH issue: #1554 

#### Implementation Details

* The docstrings are updated based on reports from [pydocstyle](http://www.pydocstyle.org/en/stable/).
* The docstrings are double-checked in API docs under `docs` directory (generated by [sphinx-autobuild](https://github.com/executablebooks/sphinx-autobuild)).

-------------------
### Test and Document the changes

1. Install `pydocstyle` using `pip install pydocstyle[toml]` or `pipx install pydocstyle[toml]`
2. Run `pydocstyle -e` (if `pydocstyle` reported any issue, please report the error)
3. In another terminal, go to `jans-pycloudlib/docs` directory and install `sphinx-autobuild` by running `pip install sphinx-autobuild` or `pipx install sphinx-autobuild`
4. Generate API docs by running `sphinx-autobuild . --watch ../jans _build/html`; visit http://127.0.0.1:8000 to see the web pages
